### PR TITLE
fix: remove trailing slash in usage of some modules & add direct import for deno

### PR DESCRIPTION
**Pull request by crowlKats**

### CODE REVIEW

File: api/src/docs.rs [hunk 1]
```diff
@@ -883,7 +883,7 @@ impl HrefResolver for DocResolver {
     let (is_main, path) = current_resolve
       .get_file()
       .map(|short_path| (short_path.is_main, &*short_path.path))
-      .unwrap_or_default();
+      .unwrap_or((true, ""));
```
Comment (marvinhagemeister) [inline api/src/docs.rs:886]: Should we add a test for this?

Comment (crowlKats) [reply api/src/docs.rs:886]: we need to rework the enture usage block infrastructure in the next few days anyways, so i think lets just land as-is

**Review by lucacasonato — APPROVED** [review]
Test would be nice, but I'm not sure we have tests for doc usage in general right now

**Comment by crowlKats** [comment]
yea adding a test for this is tricky; i'd love to set up tests infra that lets us test individual elements on our pages, but probably not soon

**Pull request merged**